tests/arm64/Tests.lst

```diff
@@ -76681,7 +76681,7 @@ RelativePath=JIT\jit64\mcc\interop\mcc_i60\mcc_i60.cmd
 WorkingDir=JIT\jit64\mcc\interop\mcc_i60
 Expected=0
 MaxAllowedDurationSeconds=600
-Categories=EXPECTED_FAIL;9462;NEW
+Categories=EXPECTED_FAIL;9462;EXCLUDED;VARAG
```

Comment:
Assume "VARAG" refers to "variable arguments."  I guess anything is fine as long as it's consistent, but I'd expect either "VARARG" or "VAARG".